_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -0,0 +1,248 @@
+
+## Background
+
+- [Required Owned Element](https://www.w3.org/TR/wai-aria-1.1/#mustContain)
+- [Owned Element](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element)
+- [HTML in ARIA](https://www.w3.org/TR/html-aria/)
+- [Implicit WAI-ARIA Semantics](https://www.w3.org/TR/wai-aria-1.1/#implicit_semantics)
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+Element `ul` with role `list` owns at least one instance of required owned element, in this case a `span` with `role` of `listitem`.
+
+```html
+<ul role="list">
+	<span role="listitem"></span>
+	<span></span>
+</ul>
+```
+
+#### Passed example 2
+
+Element `div` with role `tablist` owns an instance of required owned element, a `span` with `role` of `tab`.
+
+```html
+<div role="tablist">
+	<span role="tab"> </span>
+</div>
+```
+
+#### Passed example 3
+
+Elements with multiple nested roles, that each own at least an instance of required owned element.
+
+```html
+<table role="grid">
+	<tr role="row">
+		<span role="cell"> </span>
+	</tr>
+</table>
+```
+
+#### Passed example 4
+
+Multiple types of required owned elements are present.
+
+```html
+<ul role="menu">
+	<li></li>
+	<!-- implicit role -->
+	<li role="menuitem"></li>
+	<li role="menuitemradio"></li>
+</ul>
+```
+
+#### Passed example 5
+
+Element with role `table` owns multiple instances of required owned elements. Note that the owned elements do not have to be an immediate children of the target element. (See accessibility support notes for more information).
+
+```html
+<div role="table">
+	<div>
+		<div>
+			<div role="row">
+				<div role="cell">my cell</div>
+				<div role="cell">and another cell</div>
+			</div>
+			<div role="row">
+				<div role="cell">my cell</div>
+				<div role="cell">and another cell</div>
+			</div>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed example 6
+
+Element with role `grid` owns multiple instances of required owned elements.
+
+```html
+<div role="grid">
+	<div role="rowgroup">
+		<div role="row">
+			<div role="gridcell">1</div>
+			<div role="gridcell">Item 1</div>
+			<div role="gridcell">2 Nos.</div>
+		</div>
+	</div>
+	<div role="rowgroup">
+		<div role="row">
+			<div role="gridcell">1</div>
+			<div role="gridcell">Item 1</div>
+			<div role="gridcell">2 Nos.</div>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed example 7
+
+Element with role `list` owns an instance of required owned element which has an implicit role of `listitem`.
+
+```html
+<div role="list">
+	<li></li>
+</div>
+```
+
+#### Passed example 8
+
+Element with role `menubar` can own multiple required owned element with role `menuitem`, `menuitemcheckbox` and `menuitmeradio`, but only one is necessary.
```

Comment:
This description is "strange", since the example shows two "owned roles". With this description, I was expecting just one.